src/sqlittaaja/config.py

```diff
@@ -33,13 +33,24 @@ def parse(self, config: dict[str, Any]):
         match config.get("exercises"):
             case list(exercises_section):
 
-                def get_exercise(exercise) -> (str, str):
+                def get_exercise(exercise) -> (str, str, list[str], list[str]):
+                    must_contain = process_word_list(
+                        exercise.get("must_contain", []), "must_contain"
```

Comment:
Maybe instead of having to repeat the variable name `must_contain` here you could pass `exercise` and `"must_contain"` to the function and it will `exercise.get(...)` everything?